client/changelist/change.go

```diff
@@ -0,0 +1,46 @@
+package changelist
+
+// tufChange represents a change to a TUF repo
+type tufChange struct {
+	// Abbreviated because Go doesn't permit a field and method of the same name
+	Actn       int    `json:"action"`
+	Role       string `json:"role"`
```

Comment:
delegations mean roles have some structural rules but can be almost anything (either they're a base role, or they're prefixed with "targets/"). ChangeType could also be an enum, I'll add that.